eventing/triggers/create-triggers.adoc

```diff
@@ -1,11 +1,70 @@
 :_content-type: ASSEMBLY
-include::_attributes/common-attributes.adoc[]
 [id="create-triggers"]
 = Creating triggers
+include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-include::snippets/serverless-brokers-intro.adoc[]
+Triggers in Knative Eventing allow you to route events from a broker to a specific subscriber based on your requirements. By defining a Trigger, you can connect event producers to consumers dynamically, ensuring events are delivered to the correct destination. 
+
+This section describes the steps to create a Trigger, configure its filters, and verify its functionality. Whether you're working with simple routing needs or complex event-driven workflows. 
+
+The following examples displays common configurations for Triggers, demonstrating how to route events to Knative services or custom endpoints.
+
+.Example of routing events to a Knative Serving service
+
+The following Trigger routes all events from the default broker to the Knative Serving service named `my-service`:
+[source,yaml]
+----
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  name: my-service-trigger
+spec:
+  broker: default
+  subscriber:
+    ref:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: my-service
+----
+
+[NOTE]
+====
+Routing all events without a filter is recommended for debugging purposes. It allows you to observe and analyze all incoming events, helping identify issues or validate the flow of events through the broker before applying specific filters. To know more about filtering, see xref:../../eventing/triggers/advanced-trigger-filters.adoc#advanced-trigger-filter-overview_advanced-trigger-filters[Advanced trigger filters].
```

Comment:
Routing all events without "attribute filtering" is recommended for debugging purposes.